superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -901,7 +901,7 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
       );
     }
 
-    return [];
+    return <></>;
```

Comment:
just a style nit, but most times that I've seen components render nothing, it would be a `return null` here.

Reply:
not a blocker.

Reply:
ok let me test to make sure that this still renders the expected behavior.

Reply:
Ok no, that would render the default footer component because of the logic here:
https://github.com/apache/superset/blob/master/superset-frontend/src/components/Modal/Modal.tsx#L256